modules/ts/src/ts.cpp

```diff
@@ -684,7 +688,11 @@ void parseCustomOptions(int argc, char **argv)
 
     test_ipp_check = parser.get<bool>("test_ipp_check");
     if (!test_ipp_check)
+#ifndef WINRT
         test_ipp_check = getenv("OPENCV_IPP_CHECK") != NULL;
```

Comment:
Currently we are disabling APPCONTAINER on the postbuild step, so file system becomes available w/o standard WinRT restrictions. However using of OPENCV_TEST_DATA_PATH environment variable seems to be more common as no additional command line arguments and actions required if you already have OpenCV test environment configured on your machine.

Could you please explain what do you mean under using of `--env MY_ENV_VAR=MY_ENV_VALUE` command line argument. Is it updating of GTest code to support `--env` (that would be likely undocumented in official GTest docs) and starting of every test executable with this key?